vgen/src/main/java/io/gapi/vgen/ruby/RDocCommentFixer.java

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gapi.vgen.ruby;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Utility class for formatting python comments to follow RDoc style.
+ */
+public class RDocCommentFixer {
```

Comment:
We don't have to take action right now, but if this sort of comment fixing is going to be a common case across languages, it would be nice if we could refactor the some of this (e.g., the regular expressions and the abstract methods for replacing them) into a common base class, so that each language needs to specify only the action to be taken upon encountering each pattern.